packages/bsky/src/services/indexing/plugins/post.ts

```diff
@@ -153,8 +213,11 @@ const eventsForInsert = (obj: IndexedPost) => {
     }
   }
   const notified = new Set([obj.post.creator])
-  const ancestors = [...obj.ancestors].sort((a, b) => a.depth - b.depth)
+  const ancestors = [...(obj.ancestors ?? [])].sort((a, b) => a.depth - b.depth)
```

Comment:
out of curiosity, whats the point of the destructuring right back into an array?

Reply:
It's a habit I have since `sort()` mutates the array.  Luckily they are adding non-mutating versions of these array methods to the language 🙃

Reply:
I ended-up filtering out the `depth === 0` item up here rather than in the loop, which made the clone unnecessary 👍